component/core/src/main/java/org/exoplatform/social/core/jpa/storage/RDBMSIdentityStorageImpl.java

```diff
@@ -482,7 +482,9 @@ public Profile loadProfile(Profile profile) throws IdentityStorageException {
       profile.setId(String.valueOf(entity.getId()));
       if (entity.getAvatarFileId() != null && entity.getAvatarFileId() > 0) {
```

Comment:
Just a small question, may be even a unit test please:
`entity.getAvatarFileId() > 0` we shouldn't have an avatar generated for identities other than of type space or user. Is it the case for wallet ? If yes, could you check and fix the why + add a unit test that, by example for GroupIdentityProvider, no avatar file is generated/attached.

Reply:
> we shouldn't have an avatar generated for identities other than of type space or user. Is it the case for wallet ?
Yes for identities other than USER or space we don't have generated default avatar, 





> Is it the case for wallet ?

yes, for wallet admin identity we haven't default avatar too